operator/src/main/java/oracle/kubernetes/weblogic/domain/model/ObjectPatch.java

```diff
@@ -250,15 +247,12 @@ void addField(JsonPatchBuilder builder, String path, String newValue) {
   }
 
   static class DateTimeField<T> extends StringField<T> {
-    private static final DateTimeFormatter DATE_FORMAT = ISODateTimeFormat.dateTime();
-
-
-    DateTimeField(String name, Function<T, DateTime> getter) {
+    DateTimeField(String name, Function<T, OffsetDateTime> getter) {
       super(name, a -> toString(getter.apply(a)));
     }
 
-    private static String toString(DateTime dateTime) {
-      return Optional.ofNullable(dateTime).map(DATE_FORMAT::print).orElse(null);
+    private static String toString(OffsetDateTime dateTime) {
+      return Optional.ofNullable(dateTime).map(OffsetDateTime::toString).orElse(null);
```

Comment:
I think I did this, but double check that OffsetDateTime toString() is the correct RFC 3339 format.